### DISCUSSION TITLE
Correct link for Export button

### DISCUSSION THREAD

**Discussion by dmitriyrotaenko** [PaloAltoNetworks/docusaurus-openapi-docs#1046] [Q&A] [answered]
Hi! Thank you for this beautiful product. 

I am having a hard time configuring the `downloadUrl` property for the **Export** button but it won't work.

Here's my plugin config:

```js
    [
      'docusaurus-plugin-openapi-docs',
      {
        id: 'openapi',
        docsPluginId: 'classic',
        config: {
          "balances-api": {
            specPath: 'docs/dir-1/dir-2/openapi-specs/balances-api.yaml,
            outputDir: 'docs/dir-1/dir-2/endpoints/balances-api',
            sidebarOptions: {
              groupPathsBy: "tag",
              categoryLinkSource: "tag",
            },
            downloadUrl: '/dir-1/dir-2/openapi-specs/balances-api.yaml'
          },
        }
      }
    ],
```

When I click **Export**, the download fails. I also noticed a behavior that could be somehow connected to my issue. If I include the openspec file as a markdown link to a page, the link looks like this:

`http://localhost:3000/assets/files/balances-api-853c36dc9660d12cda27dc4d2efbe321.yaml`

There's some hash in the title. Is it possible that this hash somehow breaks the **Export** button? 

Adding an absolute URL of the file from the deployed website won't work, because the hash is changed everytime I change content of the openspec file.

What's the problem with my config and how should I fix it? Any help is insanely appreciated!

**Comment by sserrata** [comment]
Hi @dmitriyrotaenko, hard to provide guidance without know the error you're seeing but, FWIW, the Export button requires a _URL_ or _link_ to the OpenAPI spec, _not_ a local file path.

**Reply by dmitriyrotaenko** [reply]
Thank you, for a quick answer!

Can you please elaborate on what you mean by 'link to the OpenAPI spec, not a local file path?'

In `docusaurus.config.ts` file of the Open API project there's a line:

https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/61e4de7f94562eeedf5eebdfeb5fbcfa0783c2f4/demo/docusaurus.config.ts#L301

Looks like path to some local file or am I missing something?

**Reply by sserrata** [reply, marked as answer]
That's a URL path. To make it work I placed a copy of the spec under the static directory so that it is served at the / path.

**Reply by dmitriyrotaenko** [reply]
I see, thank you
